api-specs/avenir-esr.swagger.json

```
@@ -6,7 +6,7 @@
   },
   "servers": [
     {
-      "url": "http://avenirs-apisix-api:9080",
+      "url": "http://localhost:10001",
```

```suggestion
      "url": "http://avenirs-apisix-api:9080",
```
Tu peux revert ce change :+1: